### ISSUE TITLE
Applications: Docs for setup

### ISSUE THREAD

**Issue by vjeffrey** [chef/automate#1651]
**What problem are we solving?**

If I want to use the Applications Tab in beta, I can find the documentation so that I can configure Automate and my Habitat Supervisors to use the feature.

**Objective and key results.**

We should see an increase in beta users by opening up the beta and providing public documentation.

**Acceptance criteria.**

Docs exist on the public Automate docs.

We can give the docs to a user and they can be successful.

**Dependencies and blockers.**

n/a

**Evidence.**

I know in my heart that docs are important.

**Out of scope.**

These docs will still mark the feature as "beta" and the feature will continue to be hidden behind a feature flag. GA of the feature is out of scope.

**Open questions.**

n/a

Aha! Link: https://chef.aha.io/epics/A2-E-159

**Comment by vjeffrey** [comment]
@kmacgugan does this need to be broken up into individual issues, are you gonna tackle all the docs for this at once/in one pr?

**Comment by vjeffrey** [comment]
i'm gonna move this back to the epics column so we can create individual issues for this work

**Comment by vjeffrey** [comment]
removing the epic from our board since the work is done. we're just waiting on approval from product to merge
